.github/workflows/main.yml

```diff
@@ -123,7 +123,7 @@ jobs:
 
   linux-openj9:
```

Comment:
It seems there is still some inconsistency here. I'm not sure if the original intent was to run on macOS or Linux, but I think the key here, the name, and the actual OS used should all match.

Reply:
Just to provide some info. IIRC correctly, this was added due to some strange Error that appeared only on WebSphere (ie with IBM's JVM), see details on email discussion: https://lists.apache.org/thread/qmo39634fpdxnwconoo80zbczo0wy1wq

Reply:
Thanks @rubenada for pointing to the original discussion. Did you experience the issue described there on a linux server I guess, would you recommend switching to a linux-based image?

If we are not covering the specific use-case on the original OS we might even consider removing that job altogether and focus on more recent JDK versions or reduce the load on CI.

Reply:
@asolimando  IIRC the problem appeared on an Ubutu + WebSphere environment.
I agree, IMO we could consider removing this job and, and you say, focus on recent JDKs.